src/test/rmc-dependency-test/diamond-dependency/dependency3/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+pub struct Foo {
+    x: i32,
+    // This field only in this version.
+    y: i32,
+}
+
+// Export a function that takes a struct type which differs between this crate 
+// and the other vesion.
+pub fn take_foo(foo: &Foo) {
```

Comment:
Would there be an advantage to returning `foo.x + foo.y` so that upstream tests could assert on that?

Reply:
Good idea, thanks